code/package_piezo.sh

```diff
@@ -0,0 +1,3 @@
+sudo docker build -t web_app:latest .
+sudo docker tag web_app:latest host-172-16-113-64.nubes.stfc.ac.uk/piezo-resources/web_app:latest
+sudo docker push host-172-16-113-64.nubes.stfc.ac.uk/piezo-resources/web_app:latest
```

Comment:
There's no guarantee that the target IP address will stay as `host-172-16-113-64.nubes.stfc.ac.uk` forever - can we at least pull this out to one variable that gets reused?